templates/zerver/landing_nav.html

```diff
@@ -16,26 +16,94 @@
+                <div class="dropdown-label">Solutions<i class="fa fa-chevron-down"></i></div>
+                <ul class="dropdown-list solutions-options">
+                    <li>
+                        <a href="/for/companies/">Companies</a>
+                    </li>
+                    <li>
+                        <a href="/for/open-source/">Open source projects</a>
+                    </li>
+                    <li>
+                        <a href="/for/working-groups-and-communities/">Working groups and communities</a>
+                    </li>
+                </ul>
+            </div>
             <li data-on-page="plans">
-                <a href="/plans/">Plans</a>
+                <a href="/plans/">Pricing</a>
             </li>
+            <div class="dropdown nav-dropdown">
+                <div class="dropdown-label">Resources<i class="fa fa-chevron-down"></i></div>
+                <ul class="dropdown-list resources-options">
+                    <div class="dropdown-column">
+                        <li class="nav-header">For users</li>
+                        <li>
+                            <a href="/help/getting-started-with-zulip">Getting started</a>
+                        </li>
+                        <li>
+                            <a href="/help">Help center</a>
+                        </li>
+                        <li>
+                            <a href="https://zulip.readthedocs.io/en/latest/contributing/chat-zulip-org.html" target=”_blank”>
+                                Community chat
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/help/contact-support">Contact support</a>
+                        </li>
+                        <li>
+                            <a href="https://blog.zulip.com/">Blog</a>
+                        </li>
+                    </div>
+                    <div class="dropdown-column">
+                        <li class="nav-header">For administrators</li>
+                        <li>
+                            <a href="https://zulip.readthedocs.io/en/stable/production/install.html">
+                                Installing a Zulip server
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html">
+                                Upgrading a Zulip server
+                            </a>
+                        </li>
+                        <li>
+                            <a href="/help/getting-your-organization-started-with-zulip">
+                                Setting up your organization
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://github.com/zulip/zulip" target=”_blank” rel="noopener noreferer">GitHub</a>
```

Comment:
Careful!  These are curly quotes, which I don't think we ever want to use in HTML.

Reply:
Thanks for catching that one! Should be fixed now.